misc/helm-charts/operator/values.yaml

```diff
@@ -287,16 +288,16 @@ tls:
 # Storage configuration
 storage:
   storageClass:
-    # Set to false to use an existing StorageClass instead
+    # -- Set to false to use an existing StorageClass instead
```

Comment:
Do we want to also mention the official kubernetes docs w.r.t. the storage settings (at least for the `allowVolumeExpansion`, `reclaimPolicy`, and `volumeBindingMode` since they don't have descriptions?)
 https://kubernetes.io/docs/concepts/storage/storage-classes/

Reply:
Good idea, I've added a link to the Kubernetes docs in there.